.prettierignore

```diff
@@ -4,4 +4,4 @@ lib
 build
 *.css
 *.gen.tsx
-*.bs.js
+*.bs.js
```

Comment:
There are unintended line ending changes unrelated to this PR. Refer #9223 on how to revert these line ending changes.